cmd/mole/main.go

```diff
@@ -94,6 +98,17 @@ func stopDaemon(app cli.App) error {
 	if err != nil {
 		return err
 	}
+	conf, _ := storage.FindByName(app.Stop)
+	var removePath string
+	if conf == nil {
```

Comment:
Sorry, couldn't follow this logic. Why deleting only the pid file when the configuration is present? 
I think the entire instance dir should be destroyed, regardless of what is in the configuration.

Reply:
Ok, i miss that, in instance it have log and pid.
I think mole.log should be keep after stop and delete only pid.
What Should I refactor ?
1 delete all instance after stop log & pid.
2 delete only pid but keeping log.
What you think ?